tokenizer/src/main/java/edu/illinois/cs/cogcomp/nlp/tokenizer/TokenizerStateMachine.java

```diff
@@ -332,9 +332,14 @@ public void process(char token) {
                                 } else {
                                     // check for all uppercase and periods back to the start of the
                                     // word or a "-"
+                                    char nextnextChar = peek(2);
                                     if (getCurrent().isAbbr())
                                         return; // previous was upper case, acronym and word
                                                 // continues
+                                    else if (Character.isWhitespace(c) && Character.isLowerCase(nextnextChar))
```

Comment:
I think in this "else" clause, Character.isWhitespace(c) is redundant. Checking nnc to be lowercase looks good to me